compiler/common-artifacts/src/TestDataGenerator.cpp

```diff
@@ -216,8 +252,21 @@ int entry(int argc, char **argv)
       auto byte_size = dtype_size * data_size;
       std::vector<int8_t> data(byte_size);
 
+      std::string range_name;
+      int32_t range_min = 0;
+      int32_t range_max = 0;
+      if (arser["--input_range"])
+      {
+        std::vector<std::string> values = arser.get<std::vector<std::string>>("--input_range");
+        range_name = values.at(0);
+        range_min = std::atoi(values.at(1).c_str());
+        range_max = std::atoi(values.at(2).c_str());
+      }
+
       // generate random data
-      if (arser["--fixed_seed"])
+      if (range_name == input_node->name())
```

Comment:
(optional) If given name is not exist, printing it as a warning would be helpful.